src/main/java/dinkplugin/notifiers/data/LoginNotificationData.java

```diff
@@ -21,6 +21,8 @@ public class LoginNotificationData extends NotificationData {
     Progress questCount;
     Progress questPoints;
     SlayerData slayer;
+    @Nullable // requires Chat Commands plugin to be enabled
+    Map<Integer, String> petNamesById;
```

Comment:
```suggestion
    Map<Integer, String> pets;
```

Alternatively we can use a simpler name & explain in readme